releasewarrior/templates/firefox/release-rc.json.tmpl

```diff
@@ -17,9 +17,9 @@
                 "resolved": false
             },
             {
-                "alias": "beta",
+                "alias": "ship-rc",
                 "description": "publish in Balrog on beta channel",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#publish-the-release",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto.md#ship-rc",
```

Comment:
This url seems wrong. Also, seems to point to existing #howto/ship-rc when you added separate RC docs in this PR https://github.com/mozilla-releng/releasewarrior-2.0/pull/68/files#diff-77ab09a8bc720f91b95634e2562879b1R89